include/EASTL/variant.h

```diff
@@ -1374,7 +1376,7 @@ namespace eastl
 		static_assert(conjunction_v<is_same<variant_type, decay_t<Variants>>...>,
 					  "`visit` all variants passed to eastl::visit() must have the same type");
 	}
-
+	EA_RESTORE_VC_WARNING()
```

Comment:
Could this be moved up to line 1369 if we remove the `variants` name from the parameter  of `visit_static_assert_check`? I can see `variants` is unused in `visit_throw_bad_invariant_access` when exceptions are disabled so it's fine to disable the warning like this for that function but do we need to do it like this for `visit_static_assert_check` as well?

Reply:
My comment is too minor, I'll just merge this as is and I can move it up later.